odata-client-runtime/pom.xml

```diff
@@ -90,4 +91,25 @@
 
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
Where's the plugin version? As it's repeated please add a ${bundle.plugin.version} property into the parent pom.xml and use that.

Reply:
ah ok, I wondered. Either way is fine.